legend-engine-pure-code/legend-engine-pure-code-compiled-core/src/main/resources/core/pure/protocol/vX_X_X/transfers/executionPlan.pure

```diff
@@ -53,7 +53,7 @@ function meta::protocols::pure::vX_X_X::transformation::fromPureGraph::execution
             ^meta::protocols::pure::vX_X_X::metamodel::executionPlan::PureExpressionPlatformExecutionNode(
                _type='pureExp',
                resultType=$platform.resultType->transformResultType($mapping, $extensions),
-               pure = $platform.expression->meta::protocols::pure::vX_X_X::transformation::fromPureGraph::valueSpecification::transformValueSpecification([], newMap([])->cast(@Map<String, List<Any>>), $extensions)
+               pure = $platform.expression->meta::protocols::pure::vX_X_X::transformation::fromPureGraph::valueSpecification::transformValueSpecification([], newMap([]->cast(@Pair<String, List<Any>>)), $extensions)
```

Comment:
Legend Mixed support cannot infer the type arguments of the Map if cast is done after.  This cast of the pair parameter allow the proper inference of the Map type arguments.